src/lib.rs

```diff
@@ -433,7 +433,45 @@
 //! }
 //!
 //! assert_eq!(Enum::A(A).get_toto(), <A as WithConst>::TOTO);
+//! ```
+//!
+//! - Delegate fields
```

Comment:
Could you please document this also in the README? We kind of duplicate the guide both in `lib.rs` and in the README, it would be cool if it could be unified somehow.